script/time.py

```diff
@@ -67,3 +68,24 @@
 print(f"--> Time taken: {delta:.3f} microseconds per point")
 
 
+########################################################################################################################
+# PICKLE SUPPORT
+########################################################################################################################
+
+
+with open("evaluable.pk", "wb") as f:
+    pickle.dump(evaluable, f, pickle.HIGHEST_PROTOCOL)
+    
+with open("evaluable.pk", "rb") as f:
+    evaluable2 = pickle.load(f)
```

Comment:
Maybe add a minimal test doing this to check it doesn't break along the way?

Reply:
Good point; I thought an integration test in Python wouldn't hurt.

Also and especially regarding the other functional.